src/utils/config/service-helpers.js

```diff
@@ -459,6 +459,9 @@ export function cleanServiceGroups(groups) {
 
           // unifi
           site,
+
+          // speedtest-tracker
+          bitrateNumOfDecimalPlaces,
```

Comment:
```suggestion
          bitratePrecision,
```

Also this list should be alphabetical-ish

Reply:
by service widget name right?

Reply:
Yep!